rllib/examples/env/cliff_walking_wall_env.py

```diff
@@ -0,0 +1,63 @@
+import gym
```

Comment:
Can you create a unittest for this env? 
Just check that if the agent is at a few places (corner cases) the env's next state and reward are as expected when you take corner case actions.